udata/api/__init__.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, app=None, **kwargs):
         self.authorizations = {
             'apikey': {
                 'type': 'apiKey',
-                'passAs': 'header',
-                'keyname': HEADER_API_KEY
+                'in': 'header',
+                'name': HEADER_API_KEY
```

Comment:
Is that change and the one just below backward-compatible for existing API calls?